multinet/api.py

```diff
@@ -43,6 +43,17 @@ def get_workspace_tables(workspace: str, type: TableType = "all") -> Any:  # noq
     return util.stream(tables)
 
 
+@bp.route("/workspaces/<workspace>/tables", methods=["POST"])
+@use_kwargs({"table": fields.Str()})
+@swag_from("swagger/workspace_aql_tables.yaml")
+def create_aql_table(workspace: str, table: str) -> Any:
+    """Create a table from an AQL query."""
+    aql = request.data.decode()
+    table = db.create_aql_table(workspace, table, aql)
+
+    return (table, "200 OK")
```

Comment:
To be a little more consistent with the other routes
```suggestion
    return table
```